index.html

```diff
@@ -36,6 +37,121 @@ <h1><b>I’m Alexandra.<br>
                 </ul>  
             </div>
     </section> 
+    <section class="cards">
+        <div class="first_card">
+            <img src="Snapshoot Portfolio (9).jpg" alt="">
+            <h2>Tonic</h2>
+            <h3>CANOPY</h3>
+            <ul class="birth">
+                <li>Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            <p>A daily selection of privately
+                personalized reads; no accounts orsign-ups required.</p>
+            <ul  class="button">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <a><button>See project</button>
+        </div>
+
+        <div class="second_card">
+            <img src="Snapshoot Portfolio (10).jpg" alt="">
+            <h2>Multi-Post
+                   <br>
+                Stories</h2>
+            <h3>CANOPY</h3>
+            <ul class="birth">
+                <li>Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            <p>A daily selection of privately
+                personalized reads; no accounts orsign-ups required.</p>
+            <ul  class="button">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <button>See project</button>
+        </div>
+        <div class="third_card">
+            <img src="Snapshoot Portfolio (11).jpg" alt="">
+            <h2>Tonic</h2>
+            <h3>CANOPY</h3>
+            <ul class="birth">
+                <li>Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            <p>A daily selection of privately
+                personalized reads; no accounts orsign-ups required.</p>
+            <ul  class="button">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <button>See project</button>
+        </div>
+        <div class="fourth_card">
+            <img src="Snapshoot Portfolio (12).jpg" alt="">
+            <h2>Multi-Post
+                   <br>
+                Stories</h2>
+            <h3>CANOPY</h3>
+            <ul class="birth">
+                <li>Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            <p>A daily selection of privately
+                personalized reads; no accounts orsign-ups required.</p>
+            <ul  class="button">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <button>See project</button>
+        </div>
+        <section class="about">
+            <h1>About Myself</h1>
```

Comment:
- [ ] As suggested by the previous reviewer, kindly note that there should be only one h1` `in an HTML file because of semantics and it helps search engines and web crawlers easily understand the main subject of your page. I kindly suggest that you make only your main title of "I’m Alexandra Glad to see you!" an `h1` and kindly make use of headers from `h2` down to `h6` for any other headers in your HTML file.